avm/res/network/dnsresolver/outbound-endpoint/main.bicep

```
@@ -0,0 +1,44 @@
+metadata name = 'DNS Resolver Outbound Endpoint'
+metadata description = 'This module deploys a DNS Resolver Outbound Endpoint.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the Private DNS Resolver.')
+@minLength(1)
+param dnsResolverName string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@description('Required. The subnet ID of the inbound endpoint.')
+param subnetId string
+
+@description('Required. The name of the inbound endpoint.')
+param name string
+
+resource dnsResolver 'Microsoft.Network/dnsResolvers@2022-07-01' existing = {
+  name: dnsResolverName
+}
+
+resource dnsResolver_outboundEndpoint 'Microsoft.Network/dnsResolvers/outboundEndpoints@2022-07-01' = {
```

```suggestion
resource outboundEndpoint 'Microsoft.Network/dnsResolvers/outboundEndpoints@2022-07-01' = {
```
Same comment as for inbound